### ISSUE TITLE
Create an intake form

### ISSUE THREAD

**Issue by coreycaitlin** [18F/18f.gsa.gov#2100]
Per #2067, we want to create a form for collecting some basic information from leads, and to make the process of contacting us a little more predictable.

**Comment by gemfarmer** [comment]
@coreycaitlin pure html approach!

```html
<!DOCTYPE html>
<html>
<body>

<h2>Send e-mail to someone@example.com:</h2>

<form action="mailto:someone@example.com" method="post" enctype="text/plain">
Name:<br>
<input type="text" name="name"><br>
E-mail:<br>
<input type="text" name="mail"><br>
Comment:<br>
<input type="text" name="comment" size="50"><br><br>
<input type="submit" value="Send">
<input type="reset" value="Reset">
</form>

</body>
</html>
```

**Comment by gemfarmer** [comment]
@coreycaitlin update! More research and this shouldn't be a security concern. The current method that I suggested doesn't make sense though :( It opens the default mail application on a given device with populated content.

I think we want to do something like one of the following:
-  https://formspree.io/
-  https://www.npmjs.com/package/html-form-send-email-via-google-script-without-server
- build separate simple routing app that we link to the in the `action` attribute of the form.

**Comment by gemfarmer** [comment]
@gboone do you know if  formspree has come up before?

**Comment by gboone** [comment]
I do not know anything about formspree. I _suspect_ the Google Script option will be tricky because we'll have to auth it to an account. Is Mandrill worth exploring or is that more for sending a lot of email at once?

**Comment by coreycaitlin** [comment]
After some intrepid investigation by @gemfarmer, it sounds like...

- We need to make sure this isn't a general information collection, because PRA
- We need to make sure PII is handled and stored securely, by approved tools (on the approved IT standards profile)
- We don't really want to use an embedded Google form because we're concerned some agencies would not be able to access them

Other, similar forms out in the wild include:

- [The PIF intake form](https://proposal.pif.gov/)
- [IAA form prep information](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSdcP-ifAEPjk1FUK4q_hzkSmlAJaIyGx9y_4BSLrJI_ZLdVzw/viewform)

In the long run, the right answer might be to build a form that's fully integrated with Salesforce or whatever CRM-type tools we're using. If it comes up again, here's the language I drafted for this form:

![screen shot 2016-12-07 at 11 39 33 am](https://cloud.githubusercontent.com/assets/1450354/20983867/eb4f8c08-bc72-11e6-9e47-0adae02b3f07.png)

HOWEVER, for now, this doesn't seem like a very _minimum_ viable way to solve the core user need of knowing what will happen when they contact us, and making action a little easier to track.

Closing out this issue for now, and pursuing simpler ways of solving #2067.
